src/VisualStudio/Setup/VisualStudioSetup.csproj

```diff
@@ -147,6 +147,32 @@
     <NuGetPackageToIncludeInVsix Include="Microsoft.Composition" />
     <NuGetPackageToIncludeInVsix Include="System.Reflection.Metadata" />
     <NuGetPackageToIncludeInVsix Include="System.Collections.Immutable" />
+    <NuGetPackageToIncludeInVsix Include="System.Collections" />
```

Comment:
Do we now need to insert more facades to VS? That might cause red flags in VS perf tests. @KevinH-MS